### DISCUSSION TITLE
Upgrade from community to enterprise version

### DISCUSSION THREAD

**Discussion by ThalesGnimavo** [chatwoot/chatwoot#7250] [Self hosted]
Hello,
We are using the selfhosted cummunity version since 2 years and it works fine. We would like to upgrade to the enterprise version but there is no guide. 
How to install the enterprise version on our own linux server? the same command wget https://get.chatwoot.app/linux/install.sh, chmod +x install.sh could install the enterprise version?

Regards
Juste

**Comment by sojan-official** [comment]
Community edition us essentially the enterprise edition of Chatwoot with just the `enterprise` folder removed.  
By default Chatwoot enterprise edition is installed when you install using the cwctl or any other installation methods referred in our documentation. 

ref: https://www.chatwoot.com/docs/product/others/enterprise-edition/

**Comment by sojan-official** [comment]
ref: https://www.chatwoot.com/docs/self-hosted/enterprise-edition
